commcare_connect/templates/opportunity/steps.html

```
@@ -4,23 +4,23 @@
   <a class="{% if step == 'step1' %}active{% endif %}"
      id="step1-tab"
      data-toggle="tab"
-     {% if pk and org_slug %}href="{% url 'opportunity:init_edit' org_slug=org_slug opp_id=pk %}"{% endif %}>
+     {% if pk and org_slug and program and opportunity_created %}href="{% url 'program:opportunity_init_edit' org_slug=org_slug pk=program.program_id opp_id=pk %}"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)